src/models/confirmationsDao.js

```diff
@@ -0,0 +1,17 @@
+const { dataSource } = require('./dataSource');
+
+const getUserLikeById = async (userId) => {
+  const user = await dataSource.query(
+    `
+    SELECT events.*
+    FROM events
+    INNER JOIN event_reactions ON events.id = event_reactions.event_id
+    WHERE event_reactions.user_id = ? AND event_reactions.reaction_type = 'exited';`,
```

Comment:
```suggestion
    WHERE event_reactions.user_id = ? AND event_reactions.reaction_type = ?;`,
```
- reaction_type이 다른 종류가 될 수도 있기 때문에, 지금은 기대돼요 밖에 없을 지라도 백엔드에서는 확장성을 고려하여 변수로 받을 수 있도록 해두는 것이 좋습니다.
- parameter로 받아올 수 있도록 해주시고, default값을 exited로 두는 것을 추천드립니다.